test/api/v3/integration/user/POST-user_class_cast_spellId.test.js

```diff
@@ -161,6 +161,40 @@ describe('POST /user/class/cast/:spellId', () => {
       });
   });
 
+  it('Issue #12361: returns an error if chilling frost has already been cast', async () => {
+    await user.update({
+      'stats.class': 'wizard',
+      'stats.lvl': 15,
+      'stats.mp': 400,
+      'stats.buffs.streaks': true,
+    });
+    await user.sync();
+    await expect(user.post('/user/class/cast/frost'))
+      .to.eventually.be.rejected.and.eql({
+        code: 401,
+        error: 'NotAuthorized',
+        message: t('spellAlreadyCast'),
+      });
+    expect(user.stats.mp).to.equal(400);
+  });
+
+  it('Issue #12361: returns an error if stealth has already been cast', async () => {
+    await user.update({
+      'stats.class': 'rogue',
+      'stats.lvl': 15,
+      'stats.mp': 400,
+      'stats.buffs.stealth': 1,
+    });
+    await user.sync();
+    await expect(user.post('/user/class/cast/stealth'))
+      .to.eventually.be.rejected.and.eql({
+        code: 400,
+        error: 'BadRequest',
+        message: t('spellAlreadyCast'),
```

Comment:
@JalanshMunshi looks like the tests are failing because the `spellAlreadyCast` string does not exists in the original English strings (website/common/locales/en/spells.json)

Reply:
Hmm, it was running in my local machine. I will check this again.

Reply:
All checks are passing now. Please have a look at the PR and let me know if I need to change anything.

@paglias Also, I have added the string `spellAlreadyCast` to `locales/en/spells.json` and reverted the changes I made in the JSON files in my previous commit. We are now using a common string `spellAlreadyCast` instead of two separate ones for `Stealth` and `Chilling Frost` (`spellRogueStealthMaxedOut` and `spellWizardFrostAlreadyCast` respectively). Please let me know if we need to remove the old string messages from other JSON files as well. I have removed them from `locales/en/spells.json`.